tidb-cloud/terraform/backup-resource.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Backup Resource
+summary: Learn how to use backup resource
+---
+
+# Backup Resource
+
+> **Note:**
+>
+> Make sure you have set up a dedicated cluster
+
+## Create a backup with backup resource
+
```

Comment:
Please use the steps as 1, 2, 3 instead "first, then, and next" for this section.

Reply:
I will check all the doc